### PR TITLE
CLI: Update `generate` command for Github Actions

### PR DESCRIPTION
**Pull request by sdixon194**
#### Changes proposed in this Pull Request:

When creating a new github action using `jetpack generate`, we need some more info in `extra.autotagger` so it creates the "v1" tag as well as "v1.0.0". 

#### Jetpack product discussion

https://github.com/Automattic/jetpack/pull/19818#pullrequestreview-690889334

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* run `jetpack generate github-action` 
* When you're prompted for "Will this project require a mirror repo?" say Yes.
* When prompted for "Configure mirror repo to create new tags automatically (based on CHANGELOG.md)?" say Yes.
* After the project is created, check the project's composer.json and make sure that `extra.autotagger` is set to `{ "major": true }`
* Make sure non github-action types have `extra.autotagger` just set to `true`
* Answering negative to the autotagger question should set everything to false for all project types.

### CODE REVIEW

File: tools/cli/commands/generate.js [hunk 1]
```diff
@@ -446,6 +446,10 @@ function addMirrorRepo( composerJson, name, org, autotagger ) {
 	composerJson.extra.changelogger[
 		'link-template'
 	] = `https://github.com/${ org }/${ name }/compare/v\${old}...v\${new}`;
+	// Handle cases where we need more autotagger info for github action project types.
+	if ( autotagger && name.match( /^action-/ ) ) {
```
Comment (anomiex) [inline tools/cli/commands/generate.js:450]: This works, but it's too bad we can't directly look at whether the user selected "github-action".

**Review by anomiex — APPROVED** [review]

File: tools/cli/commands/generate.js (hunk 1 above)
Comment (sdixon194) [reply tools/cli/commands/generate.js:450]: Yeah, we'd have to pass that down pretty far. I could look at reworking this all if you think that would be better, but might be more trouble than it's worth. 

...also maybe I was happy to flex some regex skills I've picked up along the way.

Comment (anomiex) [reply tools/cli/commands/generate.js:450]: I don't think it's worth the trouble.

Sorry, I have a habit of complaining about things even though I'm ok with leaving them how they are. (: Maybe I shouldn't have assumed that approving would be enough of a signal that that was the case.

Comment (sdixon194) [reply tools/cli/commands/generate.js:450]: > Maybe I shouldn't have assumed that approving would be enough of a signal that that was the case.

No worries! Now I know :)

**Pull request merged**